docs/references/rest-apis/rest-deploy-api.md

```diff
@@ -0,0 +1,42 @@
+# Rest Deploy v2 API
+
+The `DeploymentRestService` APIs provides methods to manage the installed deployment packages.
+Identities with `rest.deploy` permissions can access these APIs.
+
+#### Get installed packages
+- Description: Provides the list of all the deployment packages installed and tracked by the framework.
+- Method: GET
+- API PATH: `/deploy/v2/`
+
+##### Responses
+- 200 OK status
+
+```JSON
+[{ "name": "packageName", "version": "packageVersion"}]
+```
+
+#### Install package from URL
+- Description: Installs the deployment package specified in the InstallRequest. If the request was already issued for the same InstallRequest, it returns the status of the installation process.
+- Method: POST
+- API PATH: `/deploy/v2/_install`
+
```

Comment:
Is it missing the InstallRequest format?

Reply:
Yup. Totally missed it. Fixed in 7ea13fea593ce68d001f64d2245a75db9b9113aa and 3a097296037705edf00755933297ae9853a0f7be